docusaurus.config.js

```diff
@@ -335,8 +335,11 @@ const config = {
             title: "Social Media",
             items: [
               {
-                label: "LinkedIn",
-                href: "https://www.linkedin.com/company/codeharborhub/",
+                label: "Linked",
+                href: "https://github.com/codeharborhub/codeharborhub",
+            position: "right",
+            className: "header-github-link",
+            "aria-label": "GitHub repository",
               },
               {
                 label: "YouTube",
```

Comment:
not added code in this file because you we can't need any update this file so, delete your code from `docusaurus.config.js` file

Reply:
Sorry i restored the code sorry its i m testing something else and sorry mistakenly it left behind